npm/react/README.md

```diff
@@ -94,6 +94,18 @@ See [Recipes](./docs/recipes.md) for more examples.
 }
 ```
 
+## Run
+
```

Comment:
It might be helpful to link to our official docs for [running Cypress](https://on.cypress.io/guides/getting-started/opening-the-app#Quick-Configuration) and for [component testing](https://on.cypress.io/guides/component-testing/writing-your-first-component-test).